git-hooks/pre-commit

```diff
@@ -10,7 +10,13 @@ exec 1>&2
 
 changed_files=$(git diff --cached --name-only)
 changed_go_files=$(git diff --cached --name-only | grep -E '.go$' || true)
-all_go_files=$(find -name '*.go' | grep -v /vendor/)
+
+find_cmd="find"
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
+   find_cmd='gfind'
+fi
+all_go_files=$($find_cmd -name '*.go' | grep -v /vendor/)
```

Comment:
I think this might be environment specific.  For example on my Mac I have `find` but not `gfind` - however the syntax for `find` is not quite correct since it needs to have the path explicitly included:

`find . -name '*.go'`

So I think this hook is missing the "." then it should work ok on Mac.